ubenchmark/pom.xml

```diff
@@ -63,7 +63,7 @@ POSSIBILITY OF SUCH DAMAGE.
   </dependencies>
 
   <properties>
-    <jmh.version>1.17.4</jmh.version>
+    <jmh.version>1.19</jmh.version>
```

Comment:
Unless a test in this PR is using something specific that requires a newer version of a dependency it should not be changing the versions of dependencies. That should be done separately as it's unrelated to this PR.

Reply:
I think JMH updates are fine. JMH is benchmark-only dependence, so updating to newer versions is fine from my point of view.

Reply:
Sure it's fine to upgrade. I meant it should be a separate commit to do so unless it's specifically required for a new test that needs functionality that's provided in a newer version of a dependency.  Ex: *If a PR needs a new version of JUnit to use some new assertFoo(...) method that doesn't exist in the current one.*

Otherwise, separating out the dependency version bumps makes for cleaner history and less merge conflicts in case there are multiple active PRs being worked on. Less work to review too (as it's less code to review :smile:).